src/esm_runscripts/namelists.py

```diff
@@ -491,7 +492,17 @@ def apply_iceberg_calving(config):
                     ib_num_old = 0
 
                 print(" * iceberg_dir = ", config["fesom"].get("iceberg_dir"))
-                ib_num_new = sum(1 for line in open(config["fesom"].get("iceberg_dir") + "/LON.dat"))
+                if os.path.isfile(config["fesom"].get("iceberg_dir") + "/LON.dat"):
```

Comment:
Won't change the functionality but I would do it like
```python
fesom_iceberg_dir = config["fesom"].get("iceberg_dir", None)
```
since this is used multiple times after that. This would decrease the code repetition.